third_party/musl/src/starboard/sys/stat.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <sys/stat.h>
+
+#if SB_API_VERSION < 16
+#include <fcntl.h>
+#include <time.h>
+#include "starboard/directory.h"
+
+int mkdir(const char *path, mode_t mode)
+{
+    if (SbDirectoryCreate(path)){
+        return 0;
+    }
+    return -1;
+}
+
+static SB_C_FORCE_INLINE time_t WindowsUsecToTimeT(int64_t time) {
+    int64_t posix_time = time - 11644473600000000ULL;
```

Comment:
Could you add a comment saying what this big constant is? Or make it a named constant where the name makes it clear?